v2/src/components/recipeBoxes/guides.json

```diff
@@ -0,0 +1,79 @@
+{
+  "auth_modes": [
+    {
+      "title": "Email password Login",
+      "icon": "emailPass",
+      "path": "/docs/emailpassword/introduction",
+      "img": "emailPassSS"
```

Comment:
The icons used for the recipes and addons etc are not as per designs

<img width="765" alt="Screenshot 2023-03-13 at 6 44 41 PM" src="https://user-images.githubusercontent.com/18233774/224712470-dc968bee-45e1-4935-8f41-0c5c533b58a2.png">
<img width="744" alt="Screenshot 2023-03-13 at 6 44 38 PM" src="https://user-images.githubusercontent.com/18233774/224712475-dd79eb00-b747-4a39-8d2c-1a4cf8b1e75f.png">
<img width="789" alt="Screenshot 2023-03-13 at 6 44 18 PM" src="https://user-images.githubusercontent.com/18233774/224712478-06069809-d074-4820-a8fc-5080286d184e.png">

Reply:
Didn't get this point, the icons are downloaded from the figma file

Reply:
<img width="134" alt="Screenshot 2023-03-15 at 12 51 35 PM" src="https://user-images.githubusercontent.com/18233774/225235759-a3f3d116-16c9-4ef7-8c15-a883738b6522.png">
<img width="103" alt="Screenshot 2023-03-15 at 12 51 29 PM" src="https://user-images.githubusercontent.com/18233774/225235770-95a8bf3b-6679-44f3-af85-bdac2c3b6ea5.png">
<img width="145" alt="Screenshot 2023-03-15 at 12 51 16 PM" src="https://user-images.githubusercontent.com/18233774/225235777-1a43cb9e-2a32-4d2b-8b41-90d3168dbac2.png">
<img width="176" alt="Screenshot 2023-03-15 at 12 51 11 PM" src="https://user-images.githubusercontent.com/18233774/225235782-50196e7d-1569-4628-b18c-9689327d96de.png">